app/assets/stylesheets/card.scss

```diff
@@ -54,6 +54,11 @@ Progress bar
     max-width: 300px;
   }
 
+.govuk-tag--ey {
+  width: min-content;
+  margin: 5px 0;
```

Comment:
`govuk-spacing` for consistency

Reply:
[bd535b9](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/924/commits/bd535b9a29cac3b141e54af8c8d21c30ee3e596e)